indexing-service/src/main/java/org/apache/druid/indexing/overlord/http/OverlordResource.java

```diff
@@ -947,7 +947,7 @@ public Response apply(TaskRunner taskRunner)
                           "http",
                           "host",
                           "8100",
-                          taskRunner.getTotalTaskSlotCount().getOrDefault("taskQueue", 0L).intValue(),
+                          taskRunner.getTotalTaskSlotCount().getOrDefault("_k8s_worker_category", 0L).intValue(),
```

Comment:
Could you please update the comment above to indicate why this specific value is required? Also, please check if the value can be brought out into a constant somewhere.